packages/headless/doc-parser/use-cases/commerce.ts

```diff
@@ -2,9 +2,133 @@ import {ActionLoaderConfiguration} from '../src/headless-export-resolvers/action
 import {ControllerConfiguration} from '../src/headless-export-resolvers/controller-resolver';
 import {EngineConfiguration} from '../src/headless-export-resolvers/engine-resolver';
 
-const controllers: ControllerConfiguration[] = [];
+const controllers: ControllerConfiguration[] = [
+  {
+    initializer: 'buildContext',
+    samplePaths: {},
```

Comment:
We should have some kind of ticket in CAPI project to eventually fill those examples out.

From my understanding, they are used quite a lot by implementers as quick reference/copy paste for Search Engine.
They are also useful for us, as often time it stops a ui-kit contributor to introduce breaking change unknowingly (as those examples will fail to build and block a PR).

Reply:
Created https://coveord.atlassian.net/browse/CAPI-918 to cover this!